src/main/java/edu/utdallas/davisbase/storage/TableFile.java

```
@@ -582,6 +576,18 @@ public int getnextRowId(RandomAccessFile file) throws IOException {
 		return -1;
 	}
 
+	public int getMaxtRowIdInTable(RandomAccessFile file) throws IOException {
+		try {
+			file.seek(0x01);
+
+			int rowId = file.readInt();
+			return (rowId);
+
+		} catch (Exception e) {
```

Don't catch all exceptions; that's bad practice. Actually, don't catch _any_ exceptions in this particular context.